src/block/block_read.c

```diff
@@ -229,7 +241,6 @@ __wt_block_read_off(WT_SESSION_IMPL *session, WT_BLOCK *block, WT_ITEM *buf, uin
             __wt_page_header_byteswap(buf->mem);
             return (0);
         }
-
```

Comment:
Not sure why there was a blank line here - deleted it as not part of usual code format.

Reply:
Not looking to start bikeshedding, but I think this was fine - it's normal to have "paragraph" breaks between the sections of code doing different things, and helps readability.

Reply:
@keitharnoldsmith looks like you're going to have to be the tiebreaker here ;-)

Reply:
[4e19228](https://github.com/wiredtiger/wiredtiger/pull/9838/commits/4e1922866b2d21fddb624b5ea5680e0492a44cf0)

Reply:
Lol. Thanks